testsuite/tests/apicast/policy/logging/test_date_logging.py

```diff
@@ -0,0 +1,33 @@
+"""
+Test for logging policy with custom date format
+"""
+from datetime import datetime, timezone
+
+import pytest
+
+from testsuite import rawobj
+from testsuite.capabilities import Capability
+
+pytestmark = [
+    pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-6594"),
+    pytest.mark.required_capabilities(Capability.LOGS),
+]
+
+
+@pytest.fixture(scope="module")
+def policy_settings():
+    """Customize the access logs format"""
+    return rawobj.PolicyConfig("logging", {
+
+        "custom_logging": '{{ time_local | date: "%m %d, %Y" }}'
+    })
+
+
+def test_date_logging(api_client, staging_gateway):
+    """
+    Test that the logs contain the log in the correct date format
+    """
+    api_client().get("/anything")
+    logs = staging_gateway.get_logs()
+    today = datetime.now(timezone.utc).strftime("%m %d, %Y")
+    assert today in logs
```

Comment:
You should use self-managed APIcast as if this test was run two times in a single day the previous value would be the same and also present in the logs.